opcua/common/manage_nodes.py

```diff
@@ -85,7 +85,13 @@ def create_variable(parent, *args):
     """
     nodeid, qname = _parse_add_args(*args[:2])
     val = _to_variant(*args[2:])
-    return node.Node(parent.server, _create_variable(parent.server, parent.nodeid, nodeid, qname, val, isproperty=False))
+    if len(args) > 3:
```

Comment:
This part should be move into `_to_variant` function because it can be also used for adding properties (or this doesn't make sense?)